tutorials/misc/plot_ecog.py

```diff
@@ -183,6 +157,87 @@ def animate(i, activity):
                                frames=show_power.shape[1],
                                interval=100, blit=True)
 
+###############################################################################
+# Now let's animate the raw time series data and add events related
+# to seizure onset that are marked in the annotations. This will be done
+# very similarly to the high gamma visualization
+
+# For this plot we will use a different angle (from left side)
+fig = plot_alignment(raw.info, subject='fsaverage', subjects_dir=subjects_dir,
+                     surfaces=['pial'], trans=None, coord_frame='head')
+mne.viz.set_3d_view(fig, -360, 90)
+
+xy, im = snapshot_brain_montage(fig, raw.info)
+
```

Comment:
We should separate these into two sections so that the animation lands in its own independent section
```suggestion
###############################################################################

```